PowerUp/app/src/main/java/powerup/systers/com/ListViewMaxHeight.java

```diff
@@ -0,0 +1,40 @@
+package powerup.systers.com;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.util.AttributeSet;
+import android.widget.ListView;
+
+public class ListViewMaxHeight extends ListView {
+
+    private final int maxHeight;
+
+    public ListViewMaxHeight(Context context) {
+        this(context, null);
+    }
+
+    public ListViewMaxHeight(Context context, AttributeSet attrs) {
+        this(context, attrs, 0);
+    }
+
+    public ListViewMaxHeight(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        if (attrs != null) {
+            TypedArray a = getContext().obtainStyledAttributes(attrs, R.styleable.ListViewMaxHeight);
```

Comment:
Use a better name for the style attributes like `styleAttributes` instead of `a`